### DISCUSSION TITLE
v5.2.3

### DISCUSSION THREAD

**Discussion by madeye** [shadowsocks/shadowsocks-android#2732] [General]
1. Add search tool for profiles. (#2682)
2. Add back `xchacha20-ietf-poly1305` cipher. (#2663, #2697)
3. Refined DDNS support. (#2722, #2731)
4. Minor fixes. (#2625, #2667, #2705)

Plugin library update: v2.0.1 (#2706)

<hr /><em>This discussion was created from the release <a href='https://github.com/shadowsocks/shadowsocks-android/releases/tag/v5.2.3'>v5.2.3</a>.</em>

**Comment by t-e-s-tweb** [comment]
Everything else is great but the only issue is that BBC iPlayer doesn't work which works using the same server on other apps such as v2ray and sagernet but not on this app. 

Even after setting the DNS to 0.0.0.0 some how iPlayer still detects the proxy.
